src/components/NFTPortfolio/Item.vue

```diff
@@ -100,8 +100,9 @@ export default {
     },
   },
   methods: {
-    fetchInfo() {
-      this.lazyFetchNFTClassAggregatedData();
+    async fetchInfo() {
+      await this.lazyFetchNFTClassAggregatedData(['owner']);
+      this.updateNFTOwners();
```

Comment:
if we exclude `owner` in `lazyFetchNFTClassAggregatedData `, then we can use `Promise.all` instead of blocking `await`